Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'tzinfo-data', platforms: [:mswin, :mswin64]
 gem 'rails', '~> 7.2.1'
 
 # Soft delete
-gem 'paranoia', '~> 2.4'
```

Comment:
No point having two separate commits for  
`gem 'paranoia', '~> 2.4'`  
and  
`gem 'paranoia'`

Should squash into one.